internal/handlers/common/insert.go

```diff
@@ -28,6 +30,8 @@ type InsertParams struct {
 	Collection string       `ferretdb:"collection"`
 	Ordered    bool         `ferretdb:"ordered,opt"`
 
+	DocSlice []*types.Document `ferretdb:"-"`
```

Comment:
We must not make our existing handlers worse.

Let's pass `*types.Array` to the `Insert` instead of iterator for now

Reply:
Fixed.